modules/monitoring-known-issues.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster_monitoring/managing-cluster-alerts.adoc
+
+[id="known-issues_{context}"]
+= Removing the `UsingDeprecatedAPIExtensionsV1Beta1` alert
+
+As of {product-title} 4.3.8, the `UsingDeprecatedAPIExtensionsV1Beta1` alert is no longer required. As long as your cluster version is 4.3.8 or newer, you can safely remove this rule.
+
+.Procedure
+
+. Verify that your cluster version is 4.3.8 or newer:
++
+----
+$ oc get clusterversion
+----
++
+If your cluster version is earlier than 4.3.8, upgrade before proceeding.
+
+. Review the contents of the `kube-apiserver` `PrometheusRule`:
++
+----
+$ oc get prometheusrules.monitoring.coreos.com/kube-apiserver -n openshift-kube-apiserver -oyaml
+----
++
+Find the `UsingDeprecatedAPIExtensionsV1Beta1` rule in the output.
+
+. Remove the `UsingDeprecatedAPIExtensionsV1Beta1` rule.
+
+** If `UsingDeprecatedAPIExtensionsV1Beta1` is the only rule listed in the output, remove it using the following command:
++
+----
+$ oc delete prometheusrules.monitoring.coreos.com/kube-apiserver -n openshift-kube-apiserver
+----
+
+** If there are more rules, you must manually remove only the `UsingDeprecatedAPIExtensionsV1Beta1` rule from that resource.
++
+// TODO: Using oc edit on prometheusrules.monitoring.coreos.com/kube-apiserver?
```

Comment:
@soltysh Can you confirm if they would do `oc get prometheusrules.monitoring.coreos.com/kube-apiserver -n openshift-kube-apiserver` to manually remove the `UsingDeprecatedAPIExtensionsV1Beta1` rule? If so, I'll just add this here for consistency in the procedure.

Reply:
You mean `oc edit` - yes, it should work.

Reply:
Even if users will remove more than needed, the operator will fix the missing bits back.